src/sentry/workflow_engine/models/detector.py

```diff
@@ -72,6 +72,12 @@ class Meta(OwnerModel.Meta):
             )
         ]
 
+    @property
+    def fingerprint(self) -> str:
```

Comment:
I don't think that fingerprint can be here, especially not as a property since detectors can produce multiple unique issues. Probably belongs on the DetectorHandler, I think

Reply:
👍 heh, sg. i just wanted to add the comment from that meeting.